LCM/scripts/python3/PerformRequiredConfigurationChecks.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+from imp        import load_source
+from os.path    import dirname, join, realpath, isfile
+from subprocess import PIPE, Popen
+from sys        import exc_info, exit, version_info
+from traceback  import format_exc
+from fcntl      import flock, LOCK_EX, LOCK_UN, LOCK_NB
+from OmsConfigHostHelpers import write_omsconfig_host_telemetry, write_omsconfig_host_switch_event, write_omsconfig_host_log, stop_old_host_instances
+from time       import sleep
+
+pathToCurrentScript = realpath(__file__)
+pathToCommonScriptsFolder = dirname(pathToCurrentScript)
+
+helperLibPath = join(pathToCommonScriptsFolder, 'helperlib.py')
+helperlib = load_source('helperlib', helperLibPath)
+
+operationStatusUtilityPath = join(pathToCommonScriptsFolder, 'OperationStatusUtility.py')
+operationStatusUtility = load_source('operationStatusUtility', operationStatusUtilityPath)
+
+operation = 'PerformConsistency'
+
+def main():
+    try:
+        run_perform_required_configuration_checks()
+    except SystemExit:
+        exit(exc_info()[1])
+    except Exception:
+        # Python 2.4-2.7 and 2.6-3 recognize different formats for exceptions. This methods works in all versions.
```

Comment:
necessary here? py3 shebang enabled

Reply:
Going to leave this since it works in all versions and the code is needed elsewhere